### DISCUSSION TITLE
Does Halide have any code for Schedule Visualization (other than debugging and tracing)?

### DISCUSSION THREAD

**Discussion by ZoroDerVonCodier** [halide/Halide#5528] [Ideas]
Does Halide have any code to produce the schedule visualizations shown in the tutorials such as:

https://halide-lang.org/tutorials/tutorial_lesson_05_scheduling_1.html

Is there interest in this to help debug complex scheduling?

**Comment by alexreinking** [comment]
The [HalideTraceViz](https://github.com/halide/Halide/tree/master/util) tool exists, though it is a little hard to use. There is definitely interest in making it more user-friendly.
